src/css/main.css

```diff
@@ -1358,6 +1358,54 @@ div.leaflet-top > .elevation {
     padding: 0 !important;
 }
 
+/* 
+Batch processor modal
+*/
+input:required:invalid {
+    border: 2px solid #FBEC5D;
+}
+
+select:required:invalid {
+    border: 2px solid #FBEC5D;
+}

```

Comment:
Since these are so general, I think you ended up overwriting other CSS. For example, in the Help tab the email and subject are now highlighted in a different color. I am sure there are other places this happened too. Can you make these more generalized to just the BP?